src/renderer/runner.ts

```diff
@@ -42,6 +42,8 @@ export class Runner {
     const { binaryManager, currentElectronVersion } = this.appState;
     const { version, localPath } = currentElectronVersion;
 
+    if (this.appState.isClearingConsoleOnRun)
+      this.appState.clearConsole();
```

Comment:
Dumb nit: I'd love some `{ }` around the if. We're in the process of adding `prettier` to avoid stupid style comments, but in the meantime, apologies for the nit 👯‍♂